packages/hash/api/src/model/knowledge/link.model.ts

```diff
@@ -121,14 +121,7 @@ export default class {
     const { data: link } = await graphApi.createLink(
       sourceEntityModel.entityId,
       {
-        /**
-         * @todo figure out what account ID we use here
-         *   Directly related to
-         *   https://app.asana.com/0/1202805690238892/1202883599104674/f
-         *   And may require consideration for
-         *   https://app.asana.com/0/1202805690238892/1202890446280569/f
-         */
-        ownedById: createdById,
+        ownedById,
```

Comment:
@thehabbos007 not sure what the purpose of this comment was. The `ownedById` here is just the owner of the link that's being created, just like the `ownedById` for the create entity method is the owner of the entity that's being created.

If there's something we need to figure out here, could you explain so we can update this comment to reflect that?

Apologies if I've misunderstood it's purpose all together.

Reply:
This comment seems to have found its way into another location that it was originally. I don't remember this being here :sweat_smile: 

I think it was a previous usage not having a clear accountId to use for the request. Don't think it's that relevant anymore as we do have a clear owner of the link in this situation